configtx/msp_test.go

```diff
@@ -48,7 +48,7 @@ func TestMSPConfigurationFailures(t *testing.T) {
 				badCert := &x509.Certificate{}
 				msp.RootCerts = append(msp.RootCerts, badCert)
 			},
-			expectedErr: "parsing root certs: asn1: syntax error: sequence truncated",
+			expectedErr: "parsing root certs: x509: malformed certificate",
```

Comment:
Why did that pop out in this PR? 
Was it not fixed in the PR that updated the dependencies?

Reply:
right, these changes were done in pr #56. I did rebase to include the correct expectedErr, so now it's not part of the current PR.